webapp-django/crashstats/crashstats/jinja2/crashstats/product_home.html

```diff
@@ -22,48 +22,54 @@ <h2>Crash Reports</h2>
-                                process_type='content',
-                            ) }}">
-                                Top Content Crashers
-                            </a>
-                        </li>
-                        <li>
-                            <a href="{{ url('topcrashers:topcrashers') }}?{{ make_query_string(
-                                product=product,
-                                version=version,
-                                process_type='gpu',
-                            ) }}">
-                                Top GPU Crashers
-                            </a>
-                        </li>
-                    </ul>
-                </div>
-                {% endfor %}
+                {% if versions %}
+                    {% for version in versions %}
+                    <div class="release_channel">
+                        <h4>{{product}} {{version}}</h4>
+                        <ul>
+                            <li>
+                                <a href="{{ url('topcrashers:topcrashers') }}?{{ make_query_string(
+                                    product=product,
+                                    version=version,
+                                ) }}">
+                                    Top Crashers
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{{ url('topcrashers:topcrashers') }}?{{ make_query_string(
+                                    product=product,
+                                    version=version,
+                                    process_type='plugin',
+                                ) }}">
+                                    Top Plugin Crashers
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{{ url('topcrashers:topcrashers') }}?{{ make_query_string(
+                                    product=product,
+                                    version=version,
+                                    process_type='content',
+                                ) }}">
+                                    Top Content Crashers
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{{ url('topcrashers:topcrashers') }}?{{ make_query_string(
+                                    product=product,
+                                    version=version,
+                                    process_type='gpu',
+                                ) }}">
+                                    Top GPU Crashers
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                    {% endfor %}
+                {% else %}
+                    <div class="release_channel">
+                        No version information available.
```

Comment:
If we have no version information at all, we might as well display an informative message.

Part of me was tempted to include a knock-knock joke. Something like:

Knock knock.
Who's there?
No.
No who?
NO VERSION INFORMATION AVAILABLE!

I figured as much as I think that's amusing, I should stick with this message and write the joke in the PR comments.